api/src/main/resources/org/openmrs/module/ugandaemrreports/reports/MER_TX_CURRENT_2019.json

```diff
@@ -0,0 +1,635 @@
+{
+    "resourceType": "MeasureReport",
+    "id": "MER-TX-Indicators",
```

Comment:
@dbaluku why is the id not MER-TX-CURRENT

Reply:
Done